1.3
2 cond MLP
Importing finished!!
cuda is going to be used!!
Dataset loaded!! Length (train dataset) - 6880
----------------------------------------------------------------
        Layer (type)               Output Shape         Param #
================================================================
           Flatten-1                  [-1, 256]               0
            Linear-2                   [-1, 64]          16,448
              ReLU-3                   [-1, 64]               0
            Linear-4                  [-1, 216]          14,040
================================================================
Total params: 30,488
Trainable params: 30,488
Non-trainable params: 0
----------------------------------------------------------------
Input size (MB): 0.00
Forward/backward pass size (MB): 0.00
Params size (MB): 0.12
Estimated Total Size (MB): 0.12
----------------------------------------------------------------
None
Task: v2lr, Epoch:1, Loss:0.001344              		Task: InvProb, Epoch:1, Loss:0.000439
Task: v2lr, Epoch:11, Loss:0.001171              		Task: InvProb, Epoch:11, Loss:0.000384
Task: v2lr, Epoch:21, Loss:0.001017              		Task: InvProb, Epoch:21, Loss:0.000339
Task: v2lr, Epoch:31, Loss:0.000898              		Task: InvProb, Epoch:31, Loss:0.000299
Task: v2lr, Epoch:41, Loss:0.000819              		Task: InvProb, Epoch:41, Loss:0.000272
Task: v2lr, Epoch:51, Loss:0.000770              		Task: InvProb, Epoch:51, Loss:0.000257
Task: v2lr, Epoch:61, Loss:0.000778              		Task: InvProb, Epoch:61, Loss:0.000258
Task: v2lr, Epoch:71, Loss:0.000807              		Task: InvProb, Epoch:71, Loss:0.000265
Task: v2lr, Epoch:81, Loss:0.000807              		Task: InvProb, Epoch:81, Loss:0.000265
Task: v2lr, Epoch:91, Loss:0.000770              		Task: InvProb, Epoch:91, Loss:0.000254
Task: v2lr, Epoch:100, Loss:0.000787              		Task: InvProb, Epoch:100, Loss:0.000258
Avg Test Loss: 0.0006845081676146947
Max Test Loss: 0.0024977182038128376
Min Test Loss: 3.03134074783884e-05
written to: ./models/v2lr/2_1.3.20231117032747_v2lr.pth
written to: ./models/v2lr/2_1.3.20231117032747_v2lr.pt
Traceback (most recent call last):
  File "/home/kunet.ae/100053678/eit/autoencoders/v2lr.py", line 133, in <module>
    loss_tracker.loc[id_config] = train_losses
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "/home/kunet.ae/100053678/.conda/envs/eit/lib/python3.11/site-packages/pandas/core/indexing.py", line 885, in __setitem__
    iloc._setitem_with_indexer(indexer, value, self.name)
  File "/home/kunet.ae/100053678/.conda/envs/eit/lib/python3.11/site-packages/pandas/core/indexing.py", line 1883, in _setitem_with_indexer
    self._setitem_with_indexer_missing(indexer, value)
  File "/home/kunet.ae/100053678/.conda/envs/eit/lib/python3.11/site-packages/pandas/core/indexing.py", line 2219, in _setitem_with_indexer_missing
    raise ValueError("cannot set a row with mismatched columns")
ValueError: cannot set a row with mismatched columns
